packages/cli/src/helpers.mts

```diff
@@ -53,3 +64,15 @@ export function getDirNameForCurrentFile(meta: ImportMeta): string {
 export function pathToTemplate(val: string): string {
 	return val.replace(/\{/g, '${');
 }
+
+export function b64DecodeUnicode(str: any) {
```

Comment:
is there a way to avoid the need for this decoder? few concerns:
1. it's very specific to github it seems
2. it looks like magic code.. why is it doing what it's doing?

Reply:
Github API returns bas64 encoded content, hence need to decode the content.